lib/l10nUtil.js

```diff
@@ -230,6 +230,7 @@ const defaultReplacements = [
   [/Chrome/g, 'Brave'],
   [/Google LLC. All rights reserved./g, 'The Brave Authors. All rights reserved.'],
   [/Google/g, 'Brave'],
+  [/Brave Safe Browsing/g, 'Google Safe Browsing'],
```

Comment:
This could probably be merged with the previous rule using negative-look-ahead, e.g. `Google(?! Safe Browsing?)`. But it could be argued this way is more readable, so I'm ok with it.